changelog.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+The format is based on [Keepa Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+# [Unreleased](https://github.com/purecloudlabs/softphone-vendor-headsets/compare/master...HEAD)
+
+# [v0.1.1](https://github.com/purecloudlabs/softphone-vendor-headsets/compare/d230063938501788fff660924cb1f530c1685499...c04b380d08681af57f238d1db170ebd465a3e035)
```

Comment:
I prefer to do tags over commits. See the webrtc-sdk.

Reply:
2nd